{{cookiecutter.github_repository}}/tests/conftest.py

```diff
@@ -78,4 +87,57 @@ def json(self):
                 obj=self, content_type='application/json;charset="utf-8"'
             )
 
+{%- if cookiecutter.add_graphql == "y" %}
+
+        def post_graphql(
+            self,
+            query,
+            op_name=None,
+            variables=None,
+            input_data=None,
+            graphql_url=None,
+            **extra,
+        ):
+            """Dedicated helper for posting GraphQL queries.
+
+            Args:
+                query (string)              - GraphQL query to run
+                op_name (string)            - If the query is a mutation or named query, you must
+                                              supply the op_name.  For annon queries ("{ ... }"),
+                                              should be None (default).
+                variables (dict)            - If provided, the "variables" field in GraphQL will be
+                                              set to this value.
+                input_data (dict)           - If provided, the $input variable in GraphQL will be set
+                                              to this value. If both ``input_data`` and ``variables``,
+                                              are provided, the ``input`` field in the ``variables``
+                                              dict will be overwritten with this value.
+                graphql_url (string)        - URL to graphql endpoint. Defaults to "/graphql".
+
+            Sets the `application/json` content type and json dumps the variables
+            if present.
+            """
+            if not graphql_url:
+                graphql_url = GRAPHQL_API_PATH
```

Comment:
Any reason for not providing this `GRAPHQL_API_PATH` as a default parameter to the function, instead of writing it here?

Reply:
No reason, this should have been default value to the function.